analyzer/ldap/main.zeek

```diff
@@ -9,6 +9,9 @@ export {
 	## Whether clear text passwords are captured or not.
 	option default_capture_password = F;
 
+  ## Whether to log LDAP search attributes or not.
+  option default_log_search_attributes = F;
```

Comment:
Let's add a test with this enabled.

To me it looks like this is currently broken. If I toggle this to `T` I see a very long `attributes` value for the test `protocol/ldap/basic.zeek` (in fact with the PCAP used in that test `ldap_search.log` now becomes >800MB big which seems wrong). I also see Zeek processes using excessive amount of memory. I suspect this behavior is due to search string building going haywire.

Reply:
Unfortunately I can't reproduce this on my local machine, the result of my ldap_search.log is similar to the one in the CI outputs (listing only the `sAMAccountName` attribute. What command did you use on your local machine?

Reply:
I added a test case for this that sets the attributes flag to `T` and checks if the output is a single `sAMAccountName`

Reply:
Thanks for the test.

I reinstalled my toolchain and wasn't able to reproduce the memory issue, so I suspect it was due to a faulty installation on my side.